tests/unit_tests/test_summary.py

```diff
@@ -5,7 +5,7 @@
 from argparse import Namespace
 from Bio import SeqIO
 import argparse
-from autometa import config
+from autometa.config import utilities as configutils
```

Comment:
Need a copyright notice and script explanation at the top.

Reply:
Same comments as above